### ISSUE TITLE
Issue in Carthage update for 1.3.0

### ISSUE THREAD

**Issue by djbalakrishnan** [Tealium/tealium-swift#29]
```
Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift/builder/tealium-swift.xcodeproj -scheme ios -configuration Release -derivedDataPath /Users/xxx/Library/Caches/org.carthage.CarthageKit/DerivedData/tealium-swift/1.3.0 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build (launched in /Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/n8/0tpq4bj53znfl7pjm3q1j5z40000gp/T/carthage-xcodebuild.ObowjA.log
```

```
/Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift/tealium/delegate/TealiumDelegateModule.swift:143:29: error: use of unresolved identifier 'TealiumMulticastDelegate'
    var multicastDelegate = TealiumMulticastDelegate<TealiumDelegate>()
                            ^~~~~~~~~~~~~~~~~~~~~~~~
/Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift/tealium/delegate/TealiumDelegateModule.swift:108:16: warning: value 'request' was defined but never used; consider replacing with boolean test
        if let request = request as? TealiumDisableRequest {
           ~~~~^~~~~~~~~~        ~~~
                                 is
/Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift/tealium/tagmanagement/TealiumTagManagementModule.swift:233:21: error: use of unresolved identifier 'TealiumMulticastDelegate'
    var delegates = TealiumMulticastDelegate<UIWebViewDelegate>()
                    ^~~~~~~~~~~~~~~~~~~~~~~~
/Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift/tealium/lifecycle/TealiumLifecycleModule.swift:81:20: error: use of unresolved identifier 'TealiumLifecyclePersistentData'
        let save = TealiumLifecyclePersistentData.save(lifecycle!, usingUniqueId: uniqueId)
                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift/tealium/lifecycle/TealiumLifecycleModule.swift:289:17: error: use of unresolved identifier 'TealiumLifecyclePersistentData'
        let _ = TealiumLifecyclePersistentData.save(lifecycle, usingUniqueId: uniqueId)
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/xxx/ios_prod/Carthage/Checkouts/tealium-swift/tealium/lifecycle/TealiumLifecycleModule.swift:296:34: error: use of unresolved identifier 'TealiumLifecyclePersistentData'
        if let loadedLifecycle = TealiumLifecyclePersistentData.load(uniqueId: uniqueId) {
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

** BUILD FAILED **


The following build commands failed:
	CompileSwift normal arm64
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
	CompileSwift normal armv7
	CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler
(4 failures)
```

**Comment by jalakoo** [comment]
@djbalakrishnan Thanks for catching that. Will have an update out today.

**Comment by jalakoo** [comment]
@djbalakrishnan 1.3.1 Released with updated target build settings in the builder project (used by Carthage). Please let us know if you experience any other issues.

**Comment by craigrouse** [comment]
@djbalakrishnan We have just released another update that fixes several issues (see the change log in the readme.md). Please let us know if there's anything else we can help with.

**Comment by craigrouse** [comment]
@djbalakrishnan this was fixed with 1.3.2 back in August. Please let me know if you have any further issues.
